RNTester/js/examples/KeyboardEventsExample/KeyboardEventsExample.js

```diff
@@ -54,13 +54,12 @@ class KeyEventExample extends React.Component<{}, State> {
             <View
               acceptsKeyboardFocus={true}
               enableFocusRing={true}
-              validKeysDown={['a', 'b', 'x', 'rightArrow']}
+              validKeysDown={['g', 'Esc', 'Enter', 'ArrowLeft']}
```

Comment:
can we also include combination keydown input? like enter should be differently recognized from enter+ctrl

Reply:
+1, but I don't mind if we leave that for later

---
In reply to: [570401738](https://github.com/microsoft/react-native-macos/pull/707#discussion_r570401738) [](ancestors = 570401738)

Reply:
We don't quite have that but we do pass all modifier keys back to clients along with the key press so clients can still filter by ctrl+enter vs. enter

Reply:
Sorry, late to the PRty 😉 - in RNW this is represented with the `keyUpEvents` prop, which is an array of objects that do specify modifier keys as well. Also, it uses down "Escape", which is correct according to the [standard](https://www.w3.org/TR/uievents-key/#named-key-attribute-value). The [Keyboard API doc](https://github.com/microsoft/react-native-windows/blob/master/vnext/proposals/active/keyboard-reconcile-desktop.md) actually incorrectly uses "Esc", but RNW is doing the right thing. cc @harinikmsft